.github/workflows/i18n-weekly-release.yml

```diff
@@ -0,0 +1,55 @@
+name: I18N Weekly Release
+on:
+  schedule:
+    - cron: '0 20 * * 0'
+
+jobs:
+  i18n-release:
+    name: Release
+    runs-on: ubuntu-20.04
+
+    steps:
+      # clone the repository
+      - uses: actions/checkout@v2
+      # enable dependencies caching (vendor and node_modules are wiped during build so they are ignored here)
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/composer/
+          key: ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm/
+          key: ${{ runner.os }}-npm-${{ hashFiles('package-lock.json') }}
+      # setup PHP, but without debug extensions for reasonable performance
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          tools:       composer
+          coverage:    none
+
+      - name: Build release
+        run: |
+          npm ci
+          npm run build
+
+      - name: Upload release
+        run: |
+          TAG="i18n-$(date +%y%m%d)"
+          NAME="Version for internationalization testing $TAG. Not for production."
+          BODY='This version is intended to test the internationalization strings. It should NOT be used for production.'
+
+          response=$(curl --request POST \
+            --header 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' \
+            --header 'Content-Type: application/json' \
+            --data "{\"tag_name\":\"$TAG\",\"target_commitish\":\"develop\",\"name\":\"$NAME\",\"body\":\"$BODY\",\"prerelease\":true}" \
+            https://api.github.com/repos/Automattic/woocommerce-payments/releases)
+
+          upload_url=$(echo $response | sed 's|.*"upload_url": "\([^{]*\).*|\1|')
+
+          curl --request POST \
+              --header 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' \
+              --header 'Content-Type: application/zip' \
+              --data-binary @woocommerce-payments.zip \
+              "${upload_url}?name=woocommerce-payments.zip"
+
+          curl --request POST ${{ secrets.GLOTPRESS_IMPORT_URL }}/$TAG
```

Comment:
Please add descriptive string between each command, so the action log becomes more readable.
For example, I see [the outputs of curl are not separated by new lines here is hard to read](https://github.com/Automattic/woocommerce-payments/runs/2586973896#step:7:38).

Reply:
Added descriptive strings in 19e6d45eb9ea71465eb64a523ea0bd286e974c53